<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bobinler</class>
 <widget class="QMainWindow" name="bobinler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bobin Hesaplama</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>buton_resim/bobin.png</normaloff>buton_resim/bobin.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Manyetik Geçirgenlik :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mgecirgenlik">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sarim">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kesitalan">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="uzunluk">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sarım Sayısı               :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bobin Kesit Alan        :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tel Uzunluğu             :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hesapla_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hesapla</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Endüktans                 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="enduktans_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>000 H</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
